tools/shared/utils/common/utils.cpp

```diff
@@ -437,7 +437,12 @@ collectIncludePaths(int argc, char *argv[])
       while (*cp && isspace(*cp)) {
         cp++;
       }
+      /* On Windows we have a drive letter before the first path separator. */
+#ifdef _WIN64
+      if (strlen(cp) < 3 || !isPathSeparator(*(cp+2))) {
```

Comment:
Did you consider performing a stricter check here, e.g. `!(isalpha(*cp) && *cp == ':' && isPathSeparator(*(cp+2)))`?

Reply:
yes, we could do exactly that, thanks.

Reply:
You shouldn't have removed the `strlen` check.